src/components/publishing/embed.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import styled, { StyledFunction } from "styled-components"
+
+interface EmbedProps {
+  section: any
+}
+
+interface FrameProps extends React.HTMLProps<HTMLIFrameElement> {
+  mobileHeight?: number
+  height: number
+}
+
+const iframe: StyledFunction<FrameProps & React.HTMLProps<HTMLIFrameElement>> = styled.iframe
+
+const Embed: React.SFC<EmbedProps> = props => {
+  const { url, height, mobile_height } = props.section
+  return <IFrame src={url} scrolling="no" frameBorder="0" height={height} mobileHeight={mobile_height} />
+}
+
+const IFrame = iframe`
+  width: 100%;
+  height: ${props => props.height + "px"};
+  @media (max-width: 600px) {
+    height: ${props => props.mobileHeight + "px"};
+  }
```

Comment:
You probably meant to do this?

```ts
const IFrame = iframe`
  ${props => pMedia.sm`
    height: ${props.mobileHeight}px;
  `}
`
```

Your example expects `pMedia.sm` to yield a `props` argument to functions nested inside the tagged template literal, but that’s only provided to functions nested inside the tagged template literal passed to `StyledFunction `s such as `iframe`.